scripts/src/java/org/oppia/android/scripts/SyntaxErrorHandler.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.android.scripts
+
+import org.xml.sax.ErrorHandler
+import org.xml.sax.SAXParseException
+
+/**
+ * Class for custom error handling of the parse exception thrown
+ * by the parser, to log the failure message in a cleaner way.
+ */
+class SyntaxErrorHandler(val repoPath: String) : ErrorHandler {
+  override fun warning(e: SAXParseException) {
```

Comment:
It seems that this class is specific to XML. Consider naming it: XmlSyntaxErrorHandler so that it's not confused among other error handling happening for other scripts.

This might also be a good indication that we need to do further subpackaging. What do you think about:

org.oppia.android.apps.scripts.regex
org.oppia.android.apps.scripts.xml
org.oppia.android.apps.scripts.common <-- common utilities shared between the scripts

For organizing the scripts?

Reply:
SGTM. DONE.